### DISCUSSION TITLE
How to avoid triggering onClick when dropping an element?

### DISCUSSION THREAD

**Discussion by baptisteArno** [clauderic/dnd-kit#312] [Q&A] [answered]
My element must be clickable and draggable:

    If just clicking, not dragging => trigger the click function
    If dragging => don't trigger the click function

I'd like to avoid triggering the onClick function when the element is dropped. I try to check isDragged or isSorted but they are set to false when the onClick function is fired

**Comment by clauderic** [comment, marked as answer]
This is a bug, related to #172
